test/Makefile

```diff
@@ -178,6 +178,7 @@ else
     endif
     export DFLAGS
 endif
+REQUIRED_ARGS+=$(PIC_FLAG)
```

Comment:
After the other PRs that's all that's needed to fix the test suite for `-fPIC` :)